Resources/Prototypes/Reagents/chemicals.yml

```diff
@@ -81,6 +81,22 @@
         - !type:ReagentThreshold
           min: 5
 
+- type: reagent
+  parent: Artifexium
```

Comment:
it should probably have its own flavor and physical description to reflect the fact that it's partially Glue.